src/containers/collaborations/components/CreateCollaborationModal.js

```diff
@@ -0,0 +1,161 @@
+  const [collaborationDescription, setCollaborationDescription] = useState('');
+  const [collaborationOrganizations, setCollaborationOrganizations] = useState([]);
+
+  const organizations :Map<UUID, Organization> = useSelector(selectOrganizations());
+  const createNewCollaborationRS :?RequestState = useRequestState([COLLABORATIONS, CREATE_NEW_COLLABORATION]);
+
+  const options = useMemo(() => {
+    const selectOptions = [];
+    organizations.forEach((org) => {
+      selectOptions.push({
+        label: org.title,
+        value: org.id,
+      });
+    });
+
+    return selectOptions;
+  }, [organizations]);
+
+  const handleOnChange = (orgOptions :?ReactSelectOption<Organization>[]) => {
+    if (!orgOptions) {
+      setCollaborationOrganizations([]);
+    }
+    else {
+      const orgIds = orgOptions.map((org) => org.value);
+      setCollaborationOrganizations(orgIds);
+    }
+  };
+
+  const handleOnClickPrimary = () => {
+    if (isNonEmptyString(collaborationTitle)) {
+      dispatch(
+        createNewCollaboration({
+          description: collaborationDescription,
+          name: collaborationName,
+          organizationIds: collaborationOrganizations,
+          title: collaborationTitle,
+        })
+      );
+    }
+    else {
+      setIsValidCollaborationTitle(false);
+    }
+  };
+
+  const handleOnChangeCollaborationTitle = (event :SyntheticInputEvent<HTMLInputElement>) => {
+    setCollaborationTitle(event.target.value || '');
+    setIsValidCollaborationTitle(true);
+  };
+
+  const handleOnChangeCollaborationName = (event :SyntheticInputEvent<HTMLInputElement>) => {
+    setCollaborationName(event.target.value || '');
+    setIsValidCollaborationName(true);
+  };
+
+  const handleOnChangeCollaborationDescription = (event :SyntheticInputEvent<HTMLInputElement>) => {
+    setCollaborationDescription(event.target.value || '');
+  };
+
+  const handleOnClose = () => {
+    onClose();
+    setTimeout(() => {
+      dispatch(resetRequestStates([CREATE_NEW_COLLABORATION]));
+    }, 1000);
+  };
+
+  const rsComponents = {
+    [RequestStates.STANDBY]: (
+      <ModalBody>
+        <StackGrid>
+          <span>Enter a title for this collaboration and an optional description.</span>
+          <div>
+            <Label htmlFor="new-collaboration-title">Title</Label>
+            <Input error={!isValidCollaborationTitle} onChange={handleOnChangeCollaborationTitle} />
+          </div>
+          <div>
+            <Label htmlFor="new-collaboration-name">Name</Label>
+            <Input error={!isValidCollaborationName} onChange={handleOnChangeCollaborationName} />
+          </div>
+          <div>
+            <Label htmlFor="new-collaboration-description">Description</Label>
+            <Input onChange={handleOnChangeCollaborationDescription} />
+          </div>
+          <div>
+            <Label htmlFor="new-collaboration-description">Select organizations to add to collaboration</Label>
+            <Select
```

Comment:
I believe the `htmlFor` value should be equal to the `id` attribute of the corresponding input element